live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/filter/RepoEventFilterHandler.java

```diff
@@ -42,13 +49,61 @@
 @Slf4j
 public class RepoEventFilterHandler
 {
-
     private final List<RepoEventFilterApplier> repoEventFilterAppliers;
+    private final CamelEventMapper camelEventMapper;
+
+    /**
+     * Method handles event/node filtering. When node is denied by any of filters, a custom property "DENY_NODE: true" in Camel Exchange is created (used to filter out in main route). In case of update event: - when current version of the node is allowed but previous version was denied, then the type of the event is altered to 'Created'. - when current version of the node is denied but previous version was allowed, then the type of the event is altered to 'Deleted'.
```

Comment:
Is this comment still correct? I can't see any mention of `DENY_NODE` in the code.

Reply:
Good catch - I fixed the javadoc.